pkg/plugins/policies/meshhealthcheck/api/v1alpha1/meshhealthcheck.go

```diff
@@ -100,50 +103,35 @@ type TcpHealthCheck struct {
 // the health check will be made for is an HTTP service.
 type HttpHealthCheck struct {
 	// If true the HttpHealthCheck is disabled
-	//  +optional
-	Disabled bool `json:"disabled,omitempty"`
+	Disabled *bool `json:"disabled,omitempty"`
 	// The HTTP path which will be requested during the health check
 	// (ie. /health)
-	//  +required
-	Path string `json:"path,omitempty"`
+	// +kubebuilder:default="/"
+	Path *string `json:"path,omitempty"`
 	// The list of HTTP headers which should be added to each health check
 	// request
-	//  +optional
-	RequestHeadersToAdd *[]HeaderValueOption `json:"requestHeadersToAdd,omitempty"`
+	RequestHeadersToAdd *[]HeaderValue `json:"requestHeadersToAdd,omitempty"`
 	// List of HTTP response statuses which are considered healthy
-	//  +optional
 	ExpectedStatuses *[]int32 `json:"expectedStatuses,omitempty"`
 }
 
 // GrpcHealthCheck defines gRPC configuration which will instruct the service
 // the health check will be made for is a gRPC service.
 type GrpcHealthCheck struct {
 	// If true the GrpcHealthCheck is disabled
-	//  +optional
-	Disabled bool `json:"disabled,omitempty"`
+	Disabled *bool `json:"disabled,omitempty"`
 	// Service name parameter which will be sent to gRPC service
-	//  +optional
-	ServiceName string `json:"serviceName,omitempty"`
+	ServiceName *string `json:"serviceName,omitempty"`
 	// The value of the :authority header in the gRPC health check request,
 	// by default name of the cluster this health check is associated with
-	//  +optional
-	Authority string `json:"authority,omitempty"`
-}
-
-type HeaderValueOption struct {
-	// Key/Value representation of the HTTP header
-	//  +required
-	Header *HeaderValue `json:"header,omitempty"`
-	// If true (default) the header values should be appended to already present ones
-	//  +optional
-	Append *bool `json:"append,omitempty"`
+	Authority *string `json:"authority,omitempty"`
 }
 
 type HeaderValue struct {
 	// Header name
-	//  +required
-	Key string `json:"key,omitempty"`
+	Key string `json:"key"`
 	// Header value
-	//  +required
-	Value string `json:"value,omitempty"`
+	Value string `json:"value"`
+	// If true (default) the header values should be appended to already present ones
+	Append *bool `json:"append,omitempty"`
```

Comment:
Does it make sense to do: https://github.com/kumahq/kuma/issues/5714 as a part of this?

Reply:
This PR makes `Path` optional with default value `/` https://github.com/kumahq/kuma/pull/5713/files#diff-636cc929d446f496b618a8a78665ad3621523c6a42e0d130687d08c87bca57ccR110, does it make sense?